kymatio/scattering2d/tests/test_scattering2d.py

```diff
@@ -122,6 +117,13 @@ def data(self, request):
             filter = filter[..., :1]
         return x, filter, y
 
+    if 'gpu' in devices:
+        x, filter, y = data
```

Comment:
can you not name this thing `filter` -- the syntactic highlighting suggests you might be overwriting a built-in function ...

(call it `filter_array` or something)